### PR TITLE
Additions to scripting page

### PR DESCRIPTION
**Pull request by splondike**
I fixed a couple of minor errors, mentioned that implementing actions in .talon files is deprecated, and added some examples to the .talon file section and Actions section.

### CODE REVIEW

File: unofficial_talon_docs.md [hunk 1]
```diff
@@ -129,25 +130,57 @@ Rules have a versatile syntax that is like a word based regex:
...
+```
+-
+# The following lists and captures are implemented in the knausj_talon repo
+# {user.letter} is a list mapping words like 'plex' or 'gust' to latin letters like 'x' or 'g'
+# {user.number} is a list mapping words like 'five' to number strings like '5'
+# <digits> is a capture that maps a variable length phrase like
+#   "one two three" onto an integer 123
 
-In place of an ordinary rule, you can also implement an [action](/unofficial_talon_docs#actions). In this case the rule has the form `action(NAME_OF_ACTION)`. The body syntax is the same. For example:
+# Saying "double letter plex" presses ctrl+a then inserts "x." then "x"
```
Comment (astropuffin) [inline unofficial_talon_docs.md:151]: Maybe clarify that the user has to implement these themselves in order to work? They might think that `double letter plex` would work out of the box. Or perhaps use a real example that is configured?

Comment (splondike) [reply unofficial_talon_docs.md:151]: I don't think there are any lists implemented in Talon by default (there are some defined). I have a comment at the start of this code block saying that the lists and captures used come from knausj_talon, do you reckon I should change the wording on that a bit?

**Review by astropuffin — APPROVED** [review]
LGTM

**Comment by splondike** [comment]
This is ready to merge by somebody who has permission to do so.

**Comment by 2shea** [comment]
Sorry for the delay - this definitely could have been merged a while back.

**Pull request merged**